contrib/node/src/python/pants/contrib/node/subsystems/resolvers/npm_resolver.py

```diff
@@ -69,6 +98,7 @@ def _emit_package_descriptor(self, node_task, target, results_dir, node_paths):
     dependenciesToRemove = [
       'dependencies', 'devDependencies', 'peerDependencies', 'optionalDependencies'
     ]
+    logger.error('Removing dependencies')
```

Comment:
I was spending over half a day before figuring out why the source file I specified in BUILD is different from the sources in tmp dir. So I believe this logging is important.